docs/architecture/blazor-for-web-forms-developers/project-structure.md

```diff
@@ -235,9 +235,9 @@ public class Program
 
 When a Blazor project is built, all Razor component and code files are compiled into a single assembly. Unlike ASP.NET Web Forms projects, Blazor doesn't support runtime compilation of the UI logic.
 
-## Run the app
+## Run the app with Hot Reload
 
-To run the Blazor Server app, press `F5` in Visual Studio. Blazor apps don't support runtime compilation. To see the results of code and component markup changes, rebuild and restart the app with the debugger attached. If you run without the debugger attached (`Ctrl+F5`), Visual Studio watches for file changes and restarts the app as changes are made. You manually refresh the browser as changes are made.
+To run the Blazor Server app, press `F5` in Visual Studio to run with the debugger attached, or `Ctrl+F5` to run without the debugger attached.
```

Comment:
```suggestion
To run the Blazor Server app, press <kbd>F5</kbd> in Visual Studio to run with the debugger attached, or <kbd>Ctrl</kbd> + <kbd>F5</kbd> to run without the debugger attached.
```

Please try to wrap references to keys in the [`<kbd>` element](https://developer.mozilla.org/en-US/docs/web/html/element/kbd).